Gemfile

```diff
@@ -94,6 +94,7 @@ group :development do
 
   gem 'mina'
   gem 'mina-multistage', require: false
+  gem 'bullet'
```

Comment:
It might be useful but I'd rather install it when we need it otherwise it's really easy to build up a whole bunch of "useful" gems that you never use :smile: